container/container.go

```diff
@@ -300,6 +301,36 @@ func (container *Container) GetRootResourcePath(path string) (string, error) {
 	return symlink.FollowSymlinkInScope(filepath.Join(container.Root, cleanPath), container.Root)
 }
 
+// ParseUserGrp takes `username` in the format of username, uid, username:groupname,
+// uid:gid, username:gid, or uid:groupname and parses the passwd file in the container
+// to return the ExecUser referred to by `username`.
+func (container *Container) ParseUserGrp(username string) (*user.ExecUser, error) {
+	passwdPath, err := user.GetPasswdPath()
```

Comment:
I believe this call always returns an err on windows, which means this new flag would not work for the windows builder.

Are we ok with that?